functions-methods.js

```diff
@@ -33,4 +64,23 @@
 // checkEmailValidity("tessmellink@novi.nl") geeft true - want @ en punt op de juiste plek
 // checkEmailValidity("n.eekenanovi.nl") geeft false - want geen @
 // checkEmailValidity("n.eeken@novinl.") geeft false - want de punt mag niet als laatst
-// checkEmailValidity("tessmellink@novi,nl") geeft false - want er staat een komma in
\ No newline at end of file
+// checkEmailValidity("tessmellink@novi,nl") geeft false - want er staat een komma in
+
+function checkEmailValidity(email) {
+   const containsApestaartje = email.includes("@");
+   const containsComma = email.includes(",");
+   const endsWithDot = email.endsWith(".");
+
+   if (containsApestaartje && !containsComma && !endsWithDot) {
+      console.log("true");
+   } else {
+      console.log("false");
+   }
```

Comment:
Als je de uitkomst niet zou loggen, maar terug zou geven, dan zou je de uitkomst van de expressie meteen terug kunnen geven:
```suggestion
   return containsApestaartje && !containsComma && !endsWithDot;
```

Je kunt dan de uitkomst van de functie loggen. Het bespaart je dan een paar regels code 😉